inst/include/Rcpp/Dimension.h

```diff
@@ -39,14 +39,14 @@ namespace Rcpp{
 	            dims = other.dims ;
 	        return *this ;
 	    }
-	    Dimension(const size_t& n1) : dims(1){
+	    Dimension(const int& n1) : dims(1){
```

Comment:
For coherence with R's interface, I think it's best to use `int` here. The `std::size_t` items used would still get casted to `int` later which of course would cause problems if we had a `std::size_t` which did not fit in `int`.